src/router.js

```diff
@@ -14,6 +16,11 @@ export const routes = [
     path: '/socialgraph',
     component: SocialGraph,
   },
+  {
+    name: 'Analysis',
```

Comment:
Can we show this as default page when user land on `/`? maybe a simple $router.push or replace in index